Dalamud/Interface/ItemSearchWindow.cs

```diff
@@ -87,7 +87,13 @@ class ItemSearchWindow : IDisposable
             ImGui.InputText("##searchbox", ref this.searchText, 32);
 
             var kinds = new List<string> {Loc.Localize("DalamudItemSelectAll", "All")};
-            kinds.AddRange(this.data.GetExcelSheet<ItemSearchCategory>().GetRows().Where(x => !string.IsNullOrEmpty(x.Name)).Select(x => x.Name));
+            kinds.AddRange(this.data.GetExcelSheet<ItemUICategory>().GetRows().Where(x => !string.IsNullOrEmpty(x.Name)).Select(x => x.Name));
+
+            // Fix (hack) display of hyphen in UI Category names
+            for(int i = 0; i < kinds.Count; i++){
+                kinds[i] = kinds[i].Replace("\u0002\u001F\u0001\u0003", "-");
```

Comment:
You can merge that straight into the `Select` above

Reply:
Good Point. Done with f626b12